cloud/kubernetes/helm/Chart.yaml

```diff
@@ -16,26 +16,23 @@ version: 1.0.0
 
 dependencies:
   - name: mysql
-    version: 1.6.2
-    repository: https://kubernetes-charts.storage.googleapis.com/
+    version: 8.2.3
+    repository: https://charts.bitnami.com/bitnami
     alias: mysql
     condition: mysql.enabled
     tags:
       - database
-  - name: efs-provisioner
-    version: 0.11.0
-    repository: https://kubernetes-charts.storage.googleapis.com/    
-    alias: aws-efs
-    condition: aws-efs.enabled
-    tags:
-      - fileStore
+  - name: aws-efs-csi-driver
+    version: 1.0.0
+    repository: https://kubernetes-sigs.github.io/aws-efs-csi-driver/
+    condition: efs-driver.enabled
   - name: influxdb
     version: 4.8.4
     repository: https://helm.influxdata.com/   
     alias: influxdb
     condition: influxdb.enabled
   - name: grafana
-    version: 6.1.14
+    version: 5.6.7
```

Comment:
Any reason to downgrade grafana chart from 6.1.14 to 5.6.7

Reply:
Reverted to older version